### DISCUSSION TITLE
Hosted IP addresses & LDAP

### DISCUSSION THREAD

**Discussion by pml-jcare** [snipe/snipe-it#10032] [Q&A] [answered]
Hi, we would like to integrate LDAP / AD with Snipe-IT so we don't have to maintain user base manually. Does anyone know what are public IP ranges for hosted Snipe-IT so I can limit LDAP / AD exposure to the internet by only allowing those public IP addresses to connect to our LDAP / AD? Thanks, Pawel

**Comment by uberbrady** [comment, marked as answer]
Hit us up at support@snipeitapp.com and we'll let you know what your current IP is (also do make sure to mention your subdomain).
